pages/_app.js

```diff
@@ -1,10 +1,36 @@
+import { useEffect, useState } from "react";
 import GlobalStyles from "../components/GlobalStyles";
+import recipes from "../flattened-download.json";
```

Comment:
You should have only one file with all your recipes. I am not sure, if that would be `flattened-download.json` - if so, please give the file a more descriptive name :)